db/system_keyspace.cc

```diff
@@ -99,6 +99,33 @@ namespace {
             props.enable_schema_commitlog();
         }
     });
+
+    const auto set_group0_table_options =
+        schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
+            static const std::unordered_set<sstring> tables = {
+                // scylla_local may store a replicated tombstone related to schema
+                // (see `make_group0_schema_version_mutation`), so we the repair GC mode
+                // like for the other group0 tables.
+                system_keyspace::SCYLLA_LOCAL,
```

Comment:
@tgrabiec @kbr-scylla @gleb-cloudius please check this listing in particular, if all the group0 tables are there and if all of these are actually group0 tables ... still wondering if there is even some better way or some existing flag to tell

Reply:
1. What if someone adds another one tomorrow?
2. What if it's an enterprise only goup0 table?

Reply:
We can add a check to group0 applier that all mutations are to tables with this attribute, then it will be caught during development/testing.

Reply:
Added the check now (and re-validated all the tables added here - caught another table which was missing: `system_keyspace::VIEW_BUILD_STATUS_V2`)